resources/lang/en/passwords.php

```diff
@@ -17,5 +17,5 @@
     'sent' => 'We have e-mailed your password reset link!',
     'token' => 'This password reset token is invalid.',
     'user' => "We can't find a user with that e-mail address.",
-
+    'invalid' => 'Your password does not match',
```

Comment:
That message could be:
"Your password do not [don't] match"
or
"Your passwords do not [don't] match"
```suggestion
    'invalid' => 'Your passwords do not match',
```